addons/auth_ldap/models/res_users.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.exceptions import AccessDenied
+
+from odoo import api, models, SUPERUSER_ID
+from odoo.modules.registry import RegistryManager
```

Comment:
Can you please follow good programming guideline https://wiki.haskell.org/Programming_guidelines then prefer odoo guideline  -- do it like 
```
from odoo import api, models, SUPERUSER_ID
from odoo.exceptions import AccessDenied
from odoo.modules.registry import RegistryManager
```